src/main/java/seedu/address/model/contact/Contact.java

```diff
@@ -36,7 +36,7 @@
     private final ZoomLink zoomLink;
     private final Set<Tag> tags = new HashSet<>();
     private final Set<UUID> linkedEvents = new HashSet<>();
-    private boolean isBookMarked;
+    private boolean isMarked;
```

Comment:
Why is this not set to final?

Reply:
If `isMarked` is set to final, then i cannot mark or unmark a contact using `setMarked`. Cannot change values of a final variable

Reply:
I need `setMarked` for creating contacts in `ContactBuilder`, i will create a new constructor just for `ContactBuilder` and make `isMark` final